introduction.md

```diff
@@ -0,0 +1 @@
+I'm introducing myself to the group.  My github is sunchron.  
```

Comment:
My favorite Code School path is Javascript.

 

From: Code School Kiddo [mailto:notifications@github.com] 
Sent: Thursday, February 05, 2015 11:35 AM
To: deadlyvipers/dojo_rules
Cc: sunchron
Subject: Re: [dojo_rules] intro (#3713)

 

In introduction.md <https://github.com/deadlyvipers/dojo_rules/pull/3713#discussion_r24175988> :

> @@ -0,0 +1 @@
> +I'm introducing myself to the group.  My github is sunchron.  

Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

—
Reply to this email directly or view it on GitHub <https://github.com/deadlyvipers/dojo_rules/pull/3713/files#r24175988> .  <https://github.com/notifications/beacon/AH4DsDYS4oWvFbVNXh9RiPhzQm7lrKlWks5no5MvgaJpZM4DcSW0.gif> 



Tom Rist
Supervisor of Network & Online Services
The Sun Chronicle
34 S. Main St.
Attleboro, MA 02703
(w)508-236-0391
(c)774-245-0049